src/test/java/com/kodilla/ecommercee/dto/ProductDtoTest.java

```diff
@@ -0,0 +1,59 @@
+package com.kodilla.ecommercee.dto;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static org.junit.Assert.*;
+
+@SpringBootTest
+@RunWith(SpringJUnit4ClassRunner.class)
+public class ProductDtoTest {
+
+    @Test
+    public void getId() {
+        //Given
+        ProductDto productDto = new ProductDto();

```

Comment:
tutaj coś chciałeś przetestować? czy to będzie uzupełnione w jakimś innym zadaniu?

Reply:
Tak po prawdzie zapomniałem o tym teście...